restaurant/src/containers/Orders/Orders.css

```diff
@@ -0,0 +1,83 @@
+html, body, #root, .App, .layout {
+    height: 100%;
+}
+
+body {
+    background-color: #e4e4e4;
+    margin: 0;
+}
+
+main {
+    padding: 30px;
+}
+
+.layout {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.spinner{
+    font-size: 6px;
+    display: flex;
+    justify-content: center;
+}
+
+.all-orders-window {
+    padding: 10px;
+    text-align: right;
+}
+
+.columns-titles {
+    display: flex;
+    justify-content: space-between;
+}
+
+.columns-titles {
+    display: flex;
+    color: rgb(68, 68, 68);
+}
+
+.columns-titles h3 {
+    font-weight: normal;
+    flex-grow: 1;
+    flex-shrink: 1;
+}
+
+.columns-titles .title-seperator {
+    width: 45px
+}
+
+.orders-columns {
+    display: flex;
+    flex-direction: row-reverse;
+}
+
+.orders-column {
+    width: 100%;
+}
+
+.column-seperation {
+    width: 6px;
+    min-width: 6px;
+    background-color: #cecece;
+    margin: 0 15px 0 15px;
+}
+
+.error-message {
+    text-align: center;
+}
+
+.link {
+    cursor: pointer;
+    text-decoration: underline;
+}
+
+.link:hover {
+    color: gray;
+}
+
+.layout .made-by {
+    text-align: center;
+    margin: 15px 0 5px 0;
+}
```

Comment:
You're serving the client code without any processing. How should ReactJS app ideally be deployed, what do the docs suggest before deployment? What happens when you run yarn build (or similar), what is minification/obfuscation, etc? Go over the list in order to understand the cost of the fact you're not doing that.

How would you ideally set up your client code? How would it be deployed?

What is CICD? How could you achieve it here?